dto.c

```diff
@@ -1407,7 +1406,7 @@ static void dto_memset(void *s, int c, size_t n, int *result)
 	thr_desc.pattern = memset_pattern;
 
 	/* cpu_size_fraction guaranteed to be >= 0 and < 1 */
```

Comment:
I would just update this comment to say the new bound (>= 0 and < 100 correct?)

Reply:
I've added the comment